it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/storage/GcsResourceManager.java

```diff
@@ -100,6 +130,24 @@ public static Builder builder(String bucket, String testClassName, Credentials c
     return new Builder(bucket, testClassName, credentials);
   }
 
+  public static Builder builder(String testClassName, Credentials credentials) {
+    checkArgument(!testClassName.equals(""));
+    return new Builder(null, testClassName, credentials);
+  }
+
+  public static String generateBucketName(String testClassName, String runId) {
+    Preconditions.checkNotNull(runId, "runId cannot be null");
```

Comment:
nit: I think you should static import `checkNotNull` for consistency (e.g. `checkArgument`).

Reply:
Done